vcpkg.json

```diff
@@ -1,15 +1,7 @@
 {
   "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
   "dependencies": [
-    "boost-algorithm",
-    "boost-foreach",
-    "boost-format",
-    "boost-geometry",
-    "boost-heap",
-    "boost-optional",
-    "boost-property-tree",
-    "boost-range",
-    "boost-tokenizer",
+    "boost-headers",
```

Comment:
I requested this port years ago, back then the argument against it was that upstream doesn't support this meta package itself. that changed apparently, half a year ago boostorg published a boost-headers repo. problem is that the single package ports pull in compiled dependencies by default, like boost::regex, which we don't use

Reply:
hmpf, it seems like this is not doing what I imagined it'd do: https://github.com/boostorg/headers/issues/3

Reply:
oh well, maybe there's some confusion, there's also this: https://github.com/microsoft/vcpkg/issues/44503.